src/components/NewMessage/NewMessage.vue

```diff
@@ -595,6 +673,20 @@ export default {
 			}
 		},
 
+		async handleEdit() {
+			try {
+				await editMessage({
+					token: this.token,
+					messageId: this.messageToEdit.id,
+					updatedMessage: this.text.trim()
+				})
```

Comment:
```suggestion
				const response = await editMessage({
					token: this.token,
					messageId: this.messageToEdit.id,
					updatedMessage: this.text.trim()
				})
				this.$store.dispatch('processMessage', { token: this.token, message: response.data.ocs.data })
```
If we patching the store here, it saves us from boilerplate in the MessagesList